x-pack/plugins/ml/server/routes/anomaly_detectors.ts

```diff
@@ -175,9 +176,11 @@ export function jobRoutes({ router, mlLicense }: RouteInitialization) {
     mlLicense.fullLicenseAPIGuard(async (context, request, response) => {
       try {
         const { jobId } = request.params;
+        const body: Partial<Job> = request.body;
```

Comment:
i don't think this is needed

Reply:
Removed here https://github.com/elastic/kibana/pull/70678/commits/fcb1bab689c96967cc703f7eac92810988bfbd68